Problem1/tc_even_numbers_sum.rb

```diff
@@ -0,0 +1,48 @@
+# File:  tc_even_numbers_sum.rb
+
+require_relative "even_numbers_sum.rb"
+require "test/unit"
+
+class TestEvenNumbersSum < Test::Unit::TestCase
+	
+	def test_empty
+		
+		even_numbers_sum = EvenNumbersSum.new
+		numbers = Array[]
+		
+		sum_of_even_numbers = even_numbers_sum.get_sum_of_even_numbers(numbers)		
```

Comment:
I'm not entirely thrilled with the naming here. It looks like there's a lot of repetition. When reading the code aloud it sounds something like: "even numbers sum get sum of even numbers with numbers". Something shorter might look like this: `even_numbers_adder.add(numbers)`.